web/styles/app_components.css

```diff
@@ -407,32 +407,27 @@ input.settings_text_input {
     margin: 0;
 }
 
-/* TODO: Once all layouts using this button
-   are modernized, the Font Awesome icon
-   should be replaced with a Zulip icon,
-   and its formatting should have no extra
-   space around its viewbox in SVG. */
```

Comment:
Probably wait to remove this until the last commit once all the icons are replaced?

Reply:
You should now remove this comment in the last commit!